### PR TITLE
Dev nick3

### PR DESCRIPTION
**Pull request by Anelon**
Tests have been added

### CODE REVIEW

File: src/test/java/cst438flights/controller/FlightRestControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+        getURL += "9999"; //flight ID
+
+        MockHttpServletResponse response = mvc.perform(get(getURL)).andReturn().getResponse();
+
+        assertThat(response.getStatus()).isEqualTo(HttpStatus.OK.value());
+
+        Reservation resultReservation = reservationJson.parseObject(response.getContentAsString());
+
+        assertThat(reservation).isEqualTo(resultReservation);
+    }
```
Comment (Nhinds88) [inline src/test/java/cst438flights/controller/FlightRestControllerTest.java:244]: Great work with these test in the flight rest controller

**Review by Nhinds88 — APPROVED** [review]
Looks good, just fixed the db info in application.properties to point to the JawsDB for the merge.

File: src/test/java/cst438flights/controller/CustomerRestControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,133 @@
+package cst438flights.controller;
```
Comment (espresso64) [inline src/test/java/cst438flights/controller/CustomerRestControllerTest.java:1]: Test functioning in eclipse

File: src/test/java/cst438flights/controller/FlightRestControllerTest.java [hunk 3]
```diff
@@ -0,0 +1,271 @@
+package cst438flights.controller;
```
Comment (espresso64) [inline src/test/java/cst438flights/controller/FlightRestControllerTest.java:1]: Test working locally

File: src/test/java/cst438flights/service/CustomerServiceTest.java [hunk 4]
```diff
@@ -0,0 +1,163 @@
+package cst438flights.service;
+
+import cst438flights.domain.*;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
```
Comment (espresso64) [inline src/test/java/cst438flights/service/CustomerServiceTest.java:5]: Test successful

File: src/test/java/cst438flights/service/FlightServiceTest.java [hunk 5]
```diff
@@ -0,0 +1,158 @@
+package cst438flights.service;
+
```
Comment (espresso64) [inline src/test/java/cst438flights/service/FlightServiceTest.java:2]: Test functioning as expected

**Review by espresso64 — APPROVED** [review]
Excellent work on the test! All tests work as expected in eclipse. Imported Junit 4 and 5 to build path

**Pull request merged**